tasks/apple/carthage.py

```diff
@@ -94,37 +94,19 @@ def _rome_upload_cmd(ctx, platform):
             --cache-prefix {}".format(platform, _rome_prefix(ctx))
 
 def _carthage_bootstrap_cmd(ctx):
-    # Applying Carthage build workaround to exclude Apple Silicon binaries.
-    # See https://github.com/Carthage/Carthage/issues/3019 for more details
-    fp = NamedTemporaryFile(delete=False, prefix='static.xcconfig.')
 
     xcode_version = find_xcode_version(ctx)
```

Comment:
These variable seem no longer used. Can you also clean up find_xcode_version, if possible?

Reply:
that means we don't want to print Xcode version too (2 lines below) correct?

Reply:
Yes, I don't think those logs are relevant anymore (as we no longer do carthage workaround)